packages/react/src/css/index.tsx

```diff
@@ -9,21 +9,36 @@ import type {
 import { createSetupError } from '../utils/error';
 
 /**
- * Create styles that can be re-used between components with a template literal.
+ * ## CSS
```

Comment:
We should probably move the jsdoc to be associated with the bottom export, as this function is specific to the tagged template usage. Same applies to keyframes. Though I'm not sure what it will look like in IDEs.

Reply:
Ah it looks like it needs to be on the very top one else IDE's don't pick them up.